config/schemas/config.yaml

```
@@ -65,40 +71,110 @@ $defs:
         - boolean
     properties:
       chunkLimitSize:
+        title: Chunk limit size for Fluentd buffer
+        description: |
+          Events will be written into chunks until the size of chunks become `chunkLimitSize`
+
+          [Buffering parameters](https://docs.fluentd.org/configuration/buffer-section#buffering-parameters)
         examples:
           - 50MB
         type: string
       flushInterval:
+        title: Flush interval for Fluentd buffer
+        description: |
+          Flushes the buffer each `flushInterval`, if [flushMode](#flushmode) is equal to `interval`
+
+          [Flushing parameters](https://docs.fluentd.org/configuration/buffer-section#flushing-parameters)
         examples:
           - 15m
         type: string
       flushMode:
-        examples:
+        title: Flush mode for Fluentd buffer
+        description: |
+          [Flushing parameters](https://docs.fluentd.org/configuration/buffer-section#flushing-parameters)
+        enum:
+          - lazy
           - interval
+          - immediate
+        meta:enum:
+          lazy: "Flushes/writes chunks once per timekey"
+          interval: "Flushes/writes chunks per specified time via flushInterval"
+          immediate: "Flushes/writes chunks immediately after events are appended into chunks"
         type: string
       flushThreadBurstInterval:
+        title: Flush thread burst interval for Fluentd buffer
+        description: |
+          The sleep interval (seconds) for threads between flushes when the output plugin flushes the waiting chunks to the next ones
+
+          [Flushing parameters](https://docs.fluentd.org/configuration/buffer-section#flushing-parameters)
         type: number
       flushThreadCount:
+        title: Flush thread count for Fluentd buffer
+        description: |
+          The number of threads to flush/write chunks in parallel
+
+          [Flushing parameters](https://docs.fluentd.org/configuration/buffer-section#flushing-parameters)
         type: integer
       retryForever:
+        title: Retry forever for Fluentd buffer
+        description: |
+          If true, plugin will ignore retry_timeout and retry_max_times options and retry flushing forever
```

```suggestion
          If true, plugin will ignore `retryTimeout` and `retryMaxTimes` options and retry flushing forever
```
We convert snake_case to camelCase in the value templates.